gateway/src/api/latest.rs

```diff
@@ -115,14 +123,21 @@ async fn get_project(
     responses(
         (status = 200, description = "Successfully got the projects list.", body = [shuttle_common::models::project::Response]),
         (status = 500, description = "Server internal error.")
+    ),
+    params(
+        PaginationDetails
```

Comment:
Do we need it here given we've derived for it the `IntoParams` trait?

Reply:
From my understanding the `IntoParams` trait is necessary to put it there in the first place? The `utoipa` macro is used for generating documentation so I would assume every parameter needs that trait so utoipa can generate docs for it.

Reply:
Ah indeed, it seems we still need to mention it, my bad: https://docs.rs/utoipa/latest/utoipa/attr.path.html#intoparams-type. So should we add it to the `get_deployments` route from the deployer as well?

Reply:
Indeed that's an oversight !